monkey/monkey_island/cc/resources/pba_file_upload.py

```diff
@@ -69,13 +69,14 @@ def delete(self, file_type):
             PBA_LINUX_FILENAME_PATH if file_type == "PBAlinux" else PBA_WINDOWS_FILENAME_PATH
         )
         filename = ConfigService.get_config_value(filename_path)
-        file_path = Path(env_singleton.env.get_config().data_dir_abs_path).joinpath(filename)
-        try:
-            if os.path.exists(file_path):
-                os.remove(file_path)
-            ConfigService.set_config_value(filename_path, "")
-        except OSError as e:
-            LOG.error("Can't remove previously uploaded post breach files: %s" % e)
+        if filename:
+            file_path = Path(env_singleton.env.get_config().data_dir_abs_path).joinpath(filename)
+            try:
+                if os.path.exists(file_path):
+                    os.remove(file_path)
+                ConfigService.set_config_value(filename_path, "")
```

Comment:
Should the call to `ConfigService.set_config_value()` be done in a `finally` to ensure that it happens if `os.remove()` raises an exception?